src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/P11SecretKeyFactory.java

```diff
@@ -134,6 +151,12 @@ static boolean checkUse(KeyInfo ki, KeyInfo si) {
         }
     }
 
+    static final class TLSKeyInfo extends KeyInfo {
```

Comment:
Documenting this TLSKeyInfo is to support JSSE using HKDF to derive various keys whose algorithms are named following the "TlsXXX" convention?

Reply:
Ok